### PR TITLE
Img user upload

### PR DESCRIPTION
**Pull request by alexvis**
Image upload and rspec

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -8,6 +8,7 @@ gem 'sass-rails', '~> 5.0'
 gem 'uglifier', '>= 1.3.0'
 gem 'jquery-rails'
 gem 'active_model_serializers', '<= 0.9.9'
+gem "carrierwave"
```
Comment (chrissyfancy) [inline Gemfile]: Be consistent with your single quotes vs double quotes!

File: config/application.rb [hunk 2]
```diff
@@ -10,6 +10,9 @@
 require "action_view/railtie"
 require "action_cable/engine"
 require "sprockets/railtie"
+require 'carrierwave'
+require 'carrierwave/orm/activerecord'
```
Comment (chrissyfancy) [inline config/application.rb]: Be consistent with quotes again

File: spec/features/profile_photo_spec.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+    click_button 'Sign In'
+
+    attach_file "user_profile_photo", "#{Rails.root}/spec/support/images/photo.png"
+    click_button "Upload Picture"
+
+    expect(page).to have_css("img[src*='photo.png']")
+  end
+end
```
Comment (chrissyfancy) [inline spec/features/profile_photo_spec.rb:17]: Looks good!

File: Gemfile (hunk 1 above)
Comment (JesseChamberlain) [reply Gemfile]: - completed

File: config/application.rb (hunk 2 above)
Comment (JesseChamberlain) [reply config/application.rb]: - completed

**Pull request merged**